### PR TITLE
feat: added create cozy button

### PR DESCRIPTION
**Pull request by y-lohse**
Adds a button on the public pages that invites the user to create a cozy. Easy.

### CODE REVIEW

File: src/drive/locales/en.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
       "https://play.google.com/store/apps/details?id=io.cozy.drive.mobile",
     "link-client-ios":
       "https://itunes.apple.com/us/app/cozy-drive/id1224102389?mt=8",
-    "link-client-web": "https://cozy.io/fr/try-it?from=albumsharing"
+    "link-client-web": "https://cozy.io/fr/try-it"
```
Comment (y-lohse) [inline src/drive/locales/en.json]: Not really related with this PR, sorry, but I was too lazy to do a separate PR just for this.

**Comment by y-lohse** [comment]
[(1) Créer un Cozy depuis la page publique de partage](https://trello.com/c/nky7QeNM/876-1-cr%C3%A9er-un-cozy-depuis-la-page-publique-de-partage)

File: src/components/Button/CozyHome.jsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { translate } from 'cozy-ui/react/I18n'
+import { ButtonLink } from 'cozy-ui/react'
+
+const CozyHome = ({ t, from }) => (
+  <ButtonLink
+    label={t('Share.create-cozy')}
+    href={`https://cozy.io/try-it?from=${encodeURIComponent(from)}`}
```
Comment (enguerran) [inline src/components/Button/CozyHome.jsx]: There is some convention to detect sources in links, like __utc__ query parameters, no?

File: src/components/Button/CozyHome.jsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  />
+)
+
+CozyHome.propTypes = {
+  t: PropTypes.func.isRequired,
```
Comment (enguerran) [inline src/components/Button/CozyHome.jsx]: I think you should get `t` from context rather than `props` or `props` should be at least optional with a default value in the button.

File: src/components/Button/CozyHome.jsx [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+  from: PropTypes.string
+}
+
+CozyHome.defaultProps = {
+  from: ''
```
Comment (enguerran) [inline src/components/Button/CozyHome.jsx]: If we omit `from`, then the link will still contain the query parameter `from`. 

```js
const url = `https://cozy.io/try-it${from ? `?from=${encodeURIComponent(from)}` : ''}`
```

or use the [URL API](https://developer.mozilla.org/en-US/docs/Web/API/URL).

File: src/drive/components/DownloadButton.jsx [hunk 5]
```diff
@@ -1,13 +1,20 @@
...
 import { Button } from 'cozy-ui/react'
 
-const DownloadButton = ({ label, disabled = false, onDownload, className }) => (
+const DownloadButton = ({
+  label,
+  disabled = false,
+  onDownload,
+  className,
+  theme
+}) => (
```
Comment (enguerran) [inline src/drive/components/DownloadButton.jsx]: We could be a little more flexible by writing:
```jsx
const DownloadButton = ({onDownload, ...props}) => (
  <Button
    onClick={() => props.onDownload()}
    icon="download"
    ...props
  />
)
```

File: src/drive/locales/en.json (hunk 1 above)
Comment (enguerran) [reply src/drive/locales/en.json]: Shouldn't that be `"link-client-web": "https://cozy.io/en/try-it"`

File: src/components/Button/CozyHome.jsx [hunk 6]
```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { translate } from 'cozy-ui/react/I18n'
+import { ButtonLink } from 'cozy-ui/react'
+
+const CozyHome = ({ t, from }) => (
```
Comment (enguerran) [inline src/components/Button/CozyHome.jsx]: This name does not reflect the fact that it is a button or a link.

File: src/drive/locales/en.json (hunk 1 above)
Comment (y-lohse) [reply src/drive/locales/en.json]: Good catch, it shouldn't have a language at all actually.

File: src/components/Button/CozyHome.jsx (hunk 2 above)
Comment (y-lohse) [reply src/components/Button/CozyHome.jsx]: Sorry, I don't understand what you mean?
